### ISSUE TITLE
[Bug] Whirlpool Hash causes crash

### ISSUE THREAD

**Issue by NumDeP** [veracrypt/VeraCrypt#523]
## Description 
Selecting Whirlpool as Hash Algorithm causes VC to crash.

## Steps to Reproduce

   1. Create Standard VC volume 
   2. After saving Volume Location and Name, select Next
   3. Choose any Encryption Algorithm (I tried several, not all)
   4. Select Whirlpool as Hash Algorithm
   

## VeraCrypt version: 

Version 1.24 on Linux (64-bit) (OpenSuse)

## Miscellaneous Information:
The bug report created didn't also do anything. Selecting `Yes` basically just opened up the site.

![Screenshot_20191019_133351](https://user-images.githubusercontent.com/31957540/67145340-c49afe00-f278-11e9-845b-6c4c205b0d81.png)

**Comment by idrassi** [comment]
This issue has been already reported on Sourceforge and I have found that it affects only the generic Linux installer and not the other Linux .deb and .rpm.
After analysis, I determined that it is caused by a bug in gcc 4.4.7 on CentOS 6 that is used for the build of the generic Linux installer.

I have already implemented a workaround for this bug: https://github.com/veracrypt/VeraCrypt/commit/4119521f9e7e11f6bae29963605b9acb2960fa8c

I will release a hotfix for version 1.24 that will include this fix.
